sdk/textanalytics/Azure.AI.TextAnalytics/src/TaskState.cs

```diff
@@ -19,11 +19,6 @@ public partial class TaskState
         /// <exception cref="ArgumentNullException"> <paramref name="name"/> is null. </exception>
         internal TaskState(DateTimeOffset lastUpdateDateTime, string name, JobStatus status)
```

Comment:
I am looking at the swagger and I don't see why the need of this customization to be here. What am I missing?